resources/web/wwi/texture_manager.js

```diff
@@ -0,0 +1,124 @@
+/* global THREE */
+'use strict';
+
+function TextureManager() {
```

Comment:
The `Manager` keyword is too vague in my opinion, also the method names are repeating the `texture` keyword, which we should try to avoid. Maybe we can call this class `TextureLoader` instead?